curvlinops/inverse.py

```diff
@@ -645,6 +648,12 @@ def torch_matmat(self, M_torch: ParameterMatrixType) -> ParameterMatrixType:
                     M_torch, param_pos, aaT_inv, ggT_inv
                 )
 
+            # restore original shapes
+            if "weight" in param_pos:
```

Comment:
Is there ever a case when a bias might be >2d (e.g. a CNN on 3D data or similar)? Or would a bias not be reshaped, so restoring the shape for biases is not an issue?